packages/arb-token-bridge-ui/src/hooks/useBalance.ts

```diff
@@ -88,18 +89,21 @@ const useBalance = ({ provider, walletAddress }: UseBalanceProps) => {
       } catch (error) {
         // log some extra info on sentry in case multi-caller fails
         Sentry.configureScope(function (scope) {
-          scope.setExtra('token_addresses', _addresses)
+          scope.setExtra('token_addresses', addresses)
           Sentry.captureException(error)
         })
         return {}
       }
     },
-    [provider, walletAddressLowercased]
+    []
```

Comment:
Why don't we have a dependency array here? Won't the callback fn get stale if address and chainId change?

Reply:
Those two parameters are passed as parameters to the function now